winsup/cygwin/include/cygwin/exit_process.h

```diff
@@ -40,14 +41,15 @@
  */
 
 #include <wchar.h>
+#include <math.h>
 
 #ifndef __INSIDE_CYGWIN__
 /* To help debugging via kill.exe */
 #define small_printf(...) fprintf(stderr, __VA_ARGS__)
 #endif
 
 static LPTHREAD_START_ROUTINE
-get_address_from_cygwin_console_helper(int bitness, wchar_t *function_name)
+get_address_from_cygwin_console_helper_and_inject_remote_thread(int bitness, wchar_t *function_name, int exit_code, DWORD pid)
```

Comment:
That's an awfully long function name. How about using the name `kill_via_console_helper()` instead?

Reply:
Done https://github.com/msys2/msys2-runtime/pull/31/commits/b867fed81f4a30033b57fecd71d2d5274671931e